httpserver/dns_text_records.cpp

```diff
@@ -0,0 +1,194 @@
+#include "../external/json.hpp"
+#include "dns_text_records.h"
+#include "version.h"
+#include "pow.hpp"
+#include <resolv.h>
+
+#include <boost/algorithm/string.hpp>
+
+using json = nlohmann::json;
+
+static constexpr char POW_DIFFICULTY_URL[] = "sentinel.messenger.loki.network";
+static constexpr char LATEST_VERSION_URL[] = "storage.version.loki.network";
+
+namespace loki {
+
+namespace dns {
+
+static std::string get_dns_record(const char* url, std::error_code& ec) {
+
+    std::string data;
+    unsigned char query_buffer[1024] = {};
+
+    // don't want to assume that ec has default value
+    ec = std::error_code{};
+
+    int response =
+        res_query(url, ns_c_in, ns_t_txt, query_buffer, sizeof(query_buffer));
+    
+    if (response == -1) {
+        LOKI_LOG(warn, "res_query failed while retrieving PoW");
+        ec = std::make_error_code(std::errc::bad_message);
+        return data;
+    }
+
+    ns_msg nsMsg;
+
+    if (ns_initparse(query_buffer, response, &nsMsg) == -1) {
+        LOKI_LOG(warn, "ns_initparse failed while retrieving PoW");
```

Comment:
Stale log message, as it's not PoW specific anymore?

Reply:
True, updated here and in two other logs.